printer_mods/zruncho/V0_No_Drop_Nuts/README.md

```diff
@@ -0,0 +1,59 @@
+# V0 No-Drop Nuts
+
+After struggling for hours to build the V0 bed assembly (which requires aligning 4 nuts simultaneously!), I had to do something.
+
+This mod is a whole family of printable inserts to **hold your V0 nuts in place**, inside the extrusions, and even automatically do the alignment for specific locations.
+
+They fit larger LDO extrusions, but not MakerBeamXL or OpenBeam.
+
+This will make your V0 build go happier, **guaranteed**, especially for the bed, panel mounts, and mid panel.
+
+Why?  They:
+- hold nuts in place, so you don't need realign, push, or even rotate the printer around
+- automatically align nuts for specific locations on the V0 bed
+- make it easy to align nuts in a row, like for the AB drives, since the 15mm spacing stack matches those
+
+No-Drop Nuts in place:
+![V0 bed assembly inside corner](images/z_inside_corner.jpg)
+
+No-Drop nuts in the rear side of the bed:
+![V0 bed assembly rear side](images/z_rear.jpg)
+
+Naked No-Drop Nuts, for the V0 bed:
+![For V0 bed, different kinds/sizes](images/z_drive_all.jpg)
+
+With these, you can completely assemble a v0 bed assembly in a few minutes.  
+
+In general, once you align any nut, you won't have to realign it ever, unless inserting additional nuts.
+
+The 15mm and 12mm versions are generic; use the 12mm for the front idlers on the V0, where the 15mm doesn't quite fit.
+
+## Print Instructions
+
+![all 6 in SuperSlicer](images/sliced.png)
+
+**Print Settings:** 3 perims, 0.4mm-0.5mm extrusion width, 0% infill, 3 top layers, 5 bottom layers.  Align the seam to rear edge.  Print as oriented; extrusion-channel-side down.  No overhangs past 45 degrees, so should be an easy print.
+
+And you can print these really, really fast.  Who's going to see them, anyway?
+
+BTW, you could probably remove most of the material and still have them align and work fine.  This didn't seem like a priority, since they each take maybe 90s to print.
+
+**Quantities: **Quantities are noted for the specific nut locations for a V0.  Use 2x of the 12mm for the front idlers.  You'll need 24x for the enclosure sides, plus plenty for the
+
+## Installation
+
```

Comment:
Can you add an explanation of which filenames go where?  It's not clear where `NoDropNut_Rear_Ext_Rear_Edge_x2.stl` is supposed to go for example

Reply:
Just pushed an update that should it abundantly clear what goes where.  There already was a pic, but now there's a list to help.  Also fixed the incomplete sentence typo.

Also, I clarified even more that LDO is currently the only supported extrusion type.  

And I've ordered some MakerBeamXL to test, to add to this mod, but there's no reason to wait for that.

@KiloQubit - anything else here you'd like before this can be merged?  Thanks.